course/format/classes/base.php

```diff
@@ -886,7 +917,7 @@ public function ajax_section_move() {
      * of the view script, it is not enough to change just this function. Do not forget
      * to add proper redirection.
      *
-     * @param int|stdClass|section_info $section Section object from database or just field course_sections.section
+     * @param int|stdClass|section_info|null $section Section object from database or just field course_sections.section
```

Comment:
This is just an incomplete documentation. Nothing to worry about, the logic remains the same.